molecule/certificates.yml

```diff
@@ -20,6 +20,31 @@
         state: present
       when: ansible_os_family == "Debian"
 
+    - name: Get Java Path
+      shell: dirname $(dirname $(readlink -f $(which java)))
+      register: java_path
+      when:
+        - fips_enabled|default(false) | bool
+        - ansible_os_family == "RedHat" and ansible_distribution_major_version in ['8', '9']
+
+    - name: Disable JVM level FIPS
+      lineinfile:
+        path: "{{java_path.stdout}}/conf/security/java.security"
+        search_string: 'security.useSystemPropertiesFile=true'
+        line: security.useSystemPropertiesFile=false
+        owner: root
+        group: root
+        mode: '0644'
+      when:
+        - fips_enabled|default(false) | bool
+        - ansible_os_family == "RedHat" and ansible_distribution_major_version in ['8', '9']
+
+    - name: Configure crypto policies
```

Comment:
Sorry, I'm bit confused here, why do we have steps here in molecule to enable FIPs (similar to fips-redhat.yml). For our molecule tests, we are enabling FIPS from [here](https://github.com/confluentinc/cp-ansible-tools/commit/c03ad0c36152887d542145ed9c66bc9defedf53e) , right?

Reply:
this will be used when we are having our local setup with FIPS, i.e. not created via jenkins job. If we are using jenkins job, it will simply not do anything since it's already set

Reply:
ok, got it, thanks